guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -41,20 +41,41 @@ endif::[]
 For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
 
 ifdef::katello,satellite[]
-+
 . Regenerate certificates.
 On the main {Project} server:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {certs-generate} --foreman-proxy-fqdn "myproxy.example.com" \
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
                        --certs-update-all \
-                       --certs-tar "~/myproxy.example.com-certs.tar"
+                       --certs-tar "~/{smartproxy-example-com}-certs.tar"
 ----
 +
-For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
+
+. If you use custom SSL certificates on your {smart-proxy-context} server, use the following command to regenerate the certificate tar file on the {ProjectServer}:
 +
-. Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/myproxy.example.com-certs.tar`
+[options="nowrap" subs="attributes"]
+----
+# {certs-generate} \
+--foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar "~/{smartproxy-example-com}-certs.tar" \
+--server-cert "/path/to/{smart-proxy-context}_cert.pem" \
+--server-key "/path/to/{smart-proxy-context}_key.pem" \
+--server-ca-cert "/path/to/{smart-proxy-context}_ca.pem" \
+--certs-update-server
+----
++
+For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {smart-proxy-context}] in _{InstallingSmartProxyDocTitle}_.
+
+. Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/{smartproxy-example-com}-certs.tar`
```

Comment:
```suggestion
. Copy the resulting tarball to your {SmartProxy}:
```

If you want, you can either use `_My_{smartproxy-example-com}-certs.tar_` or add `Adjust the path to the tarball.` below the code block (with a `+` above).

Using "we" is discouraged.